argus/tests/cloud/smoke.py

```diff
@@ -45,6 +45,15 @@ def _get_dhcp_value(key):
             return value.strip()
 
 
+def sort_func(arg):
```

Comment:
It can be a lambda function, but pylint will give out a warning similar to this one `Lambda may not be necessary`, now it depends if we want to go with the pylint or simply leave it as a lambda, suggestions?